### ISSUE TITLE
Workflow description does not parse as description

### ISSUE THREAD

**Issue by Arnei** [opencast/opencast-editor#296]
Why is the detail info for “publish” at the bottom of the list (makes it seem like a third option) and not next to the button? And why doesn’t “republish metadata” get detail info? - Sonia

**Comment by Arnei** [comment]
"Republish Metadata" lacks a description because Opencast did not provide one. In such a case, maybe a text like "No description available" should be shown.
